arwen/host/execution_benchmark_test.go

```diff
@@ -0,0 +1,123 @@
+package host
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/arwen-wasm-vm/arwen"
+	"github.com/ElrondNetwork/arwen-wasm-vm/config"
+	"github.com/ElrondNetwork/arwen-wasm-vm/mock"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/stretchr/testify/require"
+)
+
+var owner = []byte("owner")
+var receiver = []byte("receiver")
```

Comment:
This means that the test involves only 2 accounts. But since the benchmark's purpose is mostly to check wasmer performance (upgrade / downgrade), it should be sufficient.

Reply:
Yes, just 2 accounts should be enough for checking performance. We could add more accounts in the future, for other scenarios.